src/components/Form/FormFormats.js

```diff
@@ -69,7 +69,7 @@ const userMap = ({accountSearch}) => accountSearch.results.map(({user}) => ({
 
 const instructorSelect = (name) => (
     <Fields.DataSelect name={name} optionsMap={userMap}
-        request={SEARCH_INSTRUCTORS} noOptionsText={"No instructors available"}/>
+        request={SEARCH_INSTRUCTORS} noOptionsText="No instructors available"/>
```

Comment:
each attribute should be its own line. example:
```
<Fields.DataSelect name="name"
                   request={REQUEST}
                   noOptionsText="No options"
/> 
```

Reply:
This should be applied to everything

Reply:
Ok, added that formatting. Thanks